recipes/gensim/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "gensim" %}
+{% set version = "2.3.0" %}
+{% set sha256 = "7d0dccc7d2c576e270037949874800b7cfbc86ef081ff981483f612cd18e223f" %}
```

Comment:
Sorry, I don't catch, should we to update this two lines over creating PR here after each release?